pilot/pkg/networking/core/v1alpha3/accesslog_test.go

```diff
@@ -807,9 +822,40 @@ func TestBuildAccessLogFromTelemetry(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "default-envoy-provider",
+			ctx:  ctx,
+			meshConfig: &meshconfig.MeshConfig{
+				AccessLogEncoding: meshconfig.MeshConfig_JSON,
+				AccessLogFormat:   defaultFormatJSON,
+				ExtensionProviders: []*meshconfig.MeshConfig_ExtensionProvider{
```

Comment:
can we test that the values are overriden if we have them in extension provider still?

I guess mostly just Path? But you could also have JSON defined in AccessLogEncoding then use non-json in ExtensionProviders

Reply:
Just path I think. If extension provider has log format it s considered - there are already tests for it.